src/test/java/htmlverification/framework/component/DefectSummaryComponent.java

```diff
@@ -20,6 +20,8 @@ public class DefectSummaryComponent {
     public static final String  DANGEROUS_DEFECTS_HEADER_PARTIAL_TEXT_WELSH = "Peidiwch â gyrru nes iddo gael ei drwsio (diffygion peryglus)";
     public static final String DANGEROUS_DEFECTS_HEADER_PARTIAL_TEXT_WELSH_CVS = "Peidiwch â gyrru nes ei fod wedi cael ei atgyweirio (diffygion peryglus)";
 
+    public static final String DANGEROUS_DEFECTS_HEADER_TEXT_WELSH_CVS = "Peidiwch â gyrru nes ei fod wedi cael ei atgyweirio";
```

Comment:
Would the heading ever be this? Without the (diffygion peryglus) as on line 21.

Reply:
on the unit tests it does produce it without the (). I can try and change it though

Reply:
As far as I can tell the model always uses the one with brackets, so may be worth updating the unit test so it matches what will be produced. Thanks.